templates/partials/_navigation.html

```diff
@@ -0,0 +1,119 @@
+                <path d="M36.73 4.908a4.889 4.889 0 0 1-4.896 4.882 4.889 4.889 0 0 1-4.895-4.882A4.889 4.889 0 0 1 31.834.026a4.89 4.89 0 0 1 4.896 4.882zm-4.895-3.694a3.7 3.7 0 0 0-3.705 3.694 3.7 3.7 0 0 0 3.705 3.695 3.7 3.7 0 0 0 3.704-3.695 3.7 3.7 0 0 0-3.705-3.694z"></path>
+              </g>
+            </svg>
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+  <div class="p-navigation__row">
+    <nav class="p-navigation__nav">
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item{% if request.path.startswith('/mission') %} is-selected{% endif %}">
+          <a class="p-navigation__link" href="/about">About</a>
+        </li>
+        <li class="p-navigation__item{% if request.path.startswith('/blog') %} is-selected{% endif %}">
+          <a class="p-navigation__link" href="/blog">Blog</a>
+        </li>
+        <li class="p-navigation__item p-subnav" id="learn-link">
+          <a href="#learn-link-menu" aria-controls="learn-link-menu" class="p-subnav__toggle p-navigation__link">Learn</a>
+          <ul class="p-subnav__items" id="learn-link-menu" aria-hidden="true">
+            <li>
+              <p class="p-subnav__item is-title">Documentation</p>
+            </li>
+            <li>
+              <a href="/docs/olm" class="p-subnav__item">Charmed Operator Lifecycle Manager</a>
```

Comment:
```suggestion
              <a href="/docs" class="p-subnav__item">Charmed Operator Lifecycle Manager</a>
```

for now let's link to `/docs` we'll update when they actually do a change